app/models/offer.rb

```diff
@@ -215,7 +215,7 @@ def assign_reviewer(reviewer)
   end
 
   def send_new_offer_notification
-    PushService.new.send_notification Channel.reviewer, true, {
```

Comment:
Earlier, it was sending it over `reviewer` channel. We send native notifications to only channels with `user_` suffix and not to `reviewer` or `supervisor` channels.
Hence updated it to send over the all reviewers channels.